runtime/src/block_min_prioritization_fee_cache_update.rs

```diff
@@ -0,0 +1,14 @@
+use solana_sdk::{clock::Slot, transaction::SanitizedTransaction};
+
+pub trait BlockMinPrioritizationFeeCacheUpdate {
+    /// Update block's min prioritization fee with `txs`,
+    /// Returns updated min prioritization fee for `slot`
+    fn update_transactions<'a>(
+        &mut self,
+        slot: Slot,
+        txs: impl Iterator<Item = &'a SanitizedTransaction>,
+    ) -> Option<u64>;
+
```

Comment:
Is this trait necessary since there's only one implementation?

Reply:
Nah, not necessary, helped me to define the reader/writer interfaces during initial coding. Will remove them.

Reply:
traits have been removed